Modules/TPC/src/RatioGeneratorTPC.cxx

```diff
@@ -64,14 +64,12 @@ void RatioGeneratorTPC::update(Trigger t, framework::ServiceRegistryRef services
 
 void RatioGeneratorTPC::finalize(Trigger t, framework::ServiceRegistryRef)
 {
-  generatePlots();
   for (const auto& source : mConfig) {
     if (mRatios.count(source.nameOutputObject)) {
       getObjectsManager()->stopPublishing(source.nameOutputObject);
-      delete mRatios[source.nameOutputObject];
-      mRatios[source.nameOutputObject] = nullptr;
```

Comment:
Same here, please delete pointers in `~RatioGeneratorTPC` instead.

Reply:
Similar comment here. In the destructor it gave me issues when i stopped the task.